bitnami/kube-prometheus/Chart.yaml

```diff
@@ -46,4 +46,4 @@ maintainers:
 name: kube-prometheus
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kube-prometheus
-version: 8.16.2
+version: 8.16.3
```

Comment:
We are adding a new feature to the chart so, going by [semver](https://semver.org/) standards, we should bump the minor version instead of the patch one.